charts/galoy/values.yaml

```diff
@@ -16,6 +16,15 @@ image:
   repository: gcr.io/galoyapp/galoy
 
 deployment:
+- name: graphql-admin
+  args: ["lib/entrypoint/graphql-admin-server.js"]
+  healthz: true
+  targetPort: 4001
+  replicaCount: 1
+  podAnnotations:
+    prometheus.io/path: /swagger/metrics
```

Comment:
I think we want to get rig of swagger. we already have a lot of data from apollo studio.

Reply:
and apollo studio data is a lot more relevant to us than what swagger is providing